tempus_fugit_minecraft/main.py

```diff
@@ -486,10 +490,49 @@ def __init__(self, *args, **kwargs):
         # Instance of the model that handles the world.
         self.model = Model()
 
+        self.paused = False
+
         # The label that is displayed in the top left of the canvas.
-        self.label = pyglet.text.Label('', font_name='Arial', font_size=18,
-                                       x=10, y=self.height - 10, anchor_x='left', anchor_y='top',
-                                       color=(0, 0, 0, 255))
+        self.label = pyglet.text.Label(
+            text = '',
+            font_name='Arial',
+            font_size=18,
+            x=10,
+            y=self.height - 10,
+            anchor_x='left',
+            anchor_y='top',
+            color=(0, 0, 0, 255))
+
+        self.pause_label = pyglet.text.Label(
+            text="Paused",
+            font_name="Arial",
+            font_size=36,
+            width=100,
+            height=30,
+            x=self.width // 2,
+            y=self.height // 2,
+            anchor_x="center",
+        )
```

Comment:
The code is organized and speaking for itself. However, I have a question. Why we don't keep these `pyglet.text.label`s inside its corresponding functions?

Reply:
The labels need to be in the `__init__` function so that I can define them once and access them in my other functions.